src/System.Globalization.Calendars/tests/project.json

```diff
@@ -14,6 +14,11 @@
   "frameworks": {
     "dnxcore50": {
       "imports": "portable-net45+win8"
-    }
+    },
+    "net46": {
```

Comment:
cc @chcosta here is an example of a test project.json file that has multiple dependency sections.